modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -208,6 +216,22 @@ components:
                     readOnly: true
                     type: object
             type: object
+        ValidateRequest:
+            properties:
+                validationKeys:
+                    items:
+                        type: string
+                    type: array
+                values:
```

Comment:
`values` is plural but it's type is a single object entry. I would call it 'objectEntry' maybe?